src/interfaces/IEngine.sol

```diff
@@ -208,15 +274,52 @@ interface IEngine {
                            CREDIT MANAGEMENT
     //////////////////////////////////////////////////////////////*/
 
-    /// @notice tranfer sUSD into the engine and credit the account identified by the accountId
+    /// @notice tranfer $sUSD into the engine and
+    /// credit the account identified by the accountId
     /// @param _accountId the id of the account to credit
+    /// @param _amount the amount of $sUSD to transfer/credit
     function creditAccount(uint128 _accountId, uint256 _amount) external;
 
-    /// @notice withdraw sUSD from the engine and debit the account identified by the accountId
+    /// @notice transfer $USDC into the engine,
+    /// zap it into $sUSD, and then credit the account
+    /// identified by the accountId
+    /// @dev _amount of $USDC transferred into the
```

Comment:
**Note:** Isn't this untrue? Doesn't `Zap` always assume that `USDC.decimals <= sUSD.decimals`? And hence when using `creditAccountZap` and therefore `_zapIn`, precision is never lost? As we are going up in decimals?

Reply:
if you credit and account exactly 1 $USDC, then the account actually ends having more than 1 unit of credit.

see:

```
function test_credit_zap() public {
        uint256 decimalsFactor = 10 ** (18 - USDC.decimals());

        deal(address(USDC), ACTOR, SMALLEST_AMOUNT);

        vm.startPrank(ACTOR);

        USDC.approve(address(engine), type(uint256).max);

        uint256 preActorUSDCBalance = USDC.balanceOf(ACTOR);
        uint256 preEngineSUSDBalance = sUSD.balanceOf(address(engine));

        engine.creditAccountZap({
            _accountId: accountId,
            _amount: SMALLEST_AMOUNT
        });

        uint256 postActorUSDCBalance = USDC.balanceOf(ACTOR);
        uint256 postEngineSUSDBalance = sUSD.balanceOf(address(engine));

        vm.stopPrank();

        assert(postActorUSDCBalance == preActorUSDCBalance - SMALLEST_AMOUNT);
        assert(
            postEngineSUSDBalance
                == preEngineSUSDBalance + SMALLEST_AMOUNT * decimalsFactor
        );
        assert(engine.credit(accountId) == SMALLEST_AMOUNT * decimalsFactor);
    }
    ```

Reply:
Ah okay, I thought the statement `_amount of $USDC transferred into the engine may differ from the amount credited to the account due to precision differences (i.e. ERC-20 decimal discrepancies)` was referring to precision loss, not scaling differences.